### ISSUE TITLE
[object InputEvent] appears when trying to type in Laravel Nova.

### ISSUE THREAD

**Issue by muckypuddle-nick** [jfeid/nova-google-maps#2]
Hi, 

Whenever I edit a form in Laravel Nova, when i try to input something on the textfield for nova-google-maps i get the following appearing on screen: [object InputEvent]

My code:

.env
```
GMAPS_API_KEY="XXXXXXXXXXXXXXXXXXXXXXXXXXX"
GMAPS_DEFAULT_LATITUDE=40.27118
GMAPS_DEFAULT_LONGITUDE=22.4755078
GMAPS_DEFAULT_ZOOM=9
```
Model Resource:
```
NovaGoogleMaps::make('Exact Location')->setAttributes('location_lat', 'location_lng')->setValue(55.0, 3.0)->hideFromIndex()
```

Model Migration:
```
$table->float('location_lat', 12, 6)->nullable();
$table->float('location_lng', 12, 6)->nullable();  
```

nova-google-maps.php
```
<?php
return [
    'gmaps_api_key'     => env('GMAPS_API_KEY'),
    'default_latitude'  => env('GMAPS_DEFAULT_LATITUDE'),
    'default_longitude' => env('GMAPS_DEFAULT_LONGITUDE'),
    'default_zoom'      => env('GMAPS_DEFAULT_ZOOM', 9),
];
```
A quick video demonstrating the issue:
http://support-media.muckypuddle.me/28d90ab4b5d5

P.S. I have used the plugin for a couple of projects, so thanks for getting this plugin together!

**Comment by jfeid** [comment]
Hello,

Thanks for posting!

I am not able to replicate this issue. On a clean install Laravel 5.7, Nova 1.3.2 it works OK. What's your versions?

Some things you might need to check:

- Did you enable you API key to use the Places API in Google's Developer Console?

- Do you see calls in `https://maps.googleapis.com/maps/api/place/js/AutocompletionService.GetPredictions` in your Network tab? Do they receive a json response?

Apart from that, there is a small bug in initializing the `GMAPS_DEFAULT_LATITUDE` and `GMAPS_DEFAULT_LONGITUDE` but that is irrelevant with the issue you are describing.

**Comment by muckypuddle-nick** [comment]
Hi, 

I have gone through the checklist you have provided and since I use the plugin for some other projects, I went through and observed the correct behaviour. 

After some digging, the problem seems to be somewhere on my JS setup.  I have rolled back to a previous version on my project to verify my assumptions and the plugin works fine in there. 

Closing the issue since the plugin works as intended on a clean setup. If I get to the bottom of this and find out anything relevant to Nova/Laravel plugin conficts (Highly unlikely from what i know so far), I will update this thread. 

Thanks for the prompt response!
